io/disk/lvsetup.py

```diff
@@ -110,39 +111,22 @@ def setUp(self):
             self.device = self.disk
         else:
             if not self.lv_size:
-                self.lv_size = 1024
+                self.lv_size = 1024 * 1024 * 1024
```

Comment:
@narasimhan-v any reason for using (1024 * 1024 * 1024) ?
I mean if we need the value of this mathematical operation then we can directly use the result of above multiplication.
if we cannot use the the result value and if it is mandatory to use like in above formate then can you please add a comment here quoting the same reason?

Reply:
@narasimhan-v It is easy to understand this (3 times 1024 means it is GB) than `1073741824`, IMO.
I think the multiplied value is what needs comment, and this is easily understandable.